lib/active_admin_import/dsl.rb

```diff
@@ -31,12 +32,13 @@ module DSL
       if result.empty?
         flash[:warning] = I18n.t('active_admin_import.file_empty_error')
       else
+        if result.has_failed?
+          flash[:error] = I18n.t('active_admin_import.failed', count: result.failed.count, model: model_name, plural_model: plural_model_name, message: result.failed_message)
```

Comment:
Line is too long. [174/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.